index.js

```diff
@@ -346,23 +376,22 @@ MediaSession.prototype = extend(MediaSession.prototype, {
         self.state = 'active';
 
         self.q.push(function(qCb) {
-          self.pc.answer(self.constraints, function (err, answer) {
-            if (err) {
-              self._log('error', 'Could not create WebRTC answer', err);
-              return self.end('failed-application');
-            }
-
-            answer.jingle.contents.forEach(filterUnusedLabels);
-            // this isn't needed current because we are signaling a source-remove and then source-add when adding a stream
-            // leaving here since the source-remove, source-add solution breaks firefox -> chrome
-            // answer.jingle.contents.forEach(filterOutRecvonly);
-            answer.jingle.contents.forEach(changeSendersIfNoMsids);
+            self.pc.answer(self.constraints, function (err, answer) {
+                if (err) {
+                    self._log('error', 'Could not create WebRTC answer', err);
+                    return self.end('failed-application');
+                }
 
-            self.send('session-accept', answer.jingle);
+                answer.jingle.contents.forEach(filterUnusedLabels);
+                // this isn't needed current because we are signaling a source-remove and then source-add when adding a stream
```

Comment:
you can remove these two comments and the commented out line

Reply:
removed